Example/src/screens/Menus.tsx

```diff
@@ -5,7 +5,7 @@ import { Platform, Text, TouchableOpacity, View } from 'react-native';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
 import { UIConstant } from '@tonlabs/uikit.core';
-import { UIPopover, UIPopoverMenu } from '@tonlabs/uikit.navigation_legacy';
+import { UIPopover } from '@tonlabs/uikit.navigation_legacy';
```

Comment:
Do you think UIPopover is still being used? :)

Reply:
Oh, true! It'd be great to kill it completely ;)